Public/Start-Polaris.ps1

```diff
@@ -44,8 +41,28 @@ function Start-Polaris {
         [switch]
         $UseJsonBodyParserMiddleware = $False,
 
+        [ValidateScript( {
+                if ([System.Environment]::OSVersion.Platform -ne [System.PlatformID]::Win32NT) {
+                    throw "SSL is not supported on Linux and Mac. Please proxy the traffic."
+                }
+                else {
+                    $true
+                }               
+            })]            
         [switch]
-        $Https = $False,        
+        $Https = $False,
+
+        [ValidateSet('Anonymous', 'Basic', 'Digest', 'IntegratedWindowsAuthentication', 'Negotiate', 'NTLM')]
+        [ValidateScript( {
+                if ([System.Environment]::OSVersion.Platform -ne [System.PlatformID]::Win32NT -and !($_ -eq 'Basic' -or $_ -eq 'Anonymous')) {
```

Comment:
```suggestion
                if (($IsMacOS -or $IsLinux) -and !($_ -in @( 'Basic', 'Anonymous'))) {
```
That should be a bit cleaner :) but I haven't tested it (except with my mind lol)